lib/msgr.rb

```diff
@@ -34,7 +34,11 @@ def config
     end
 
     def client
-      @client ||= Msgr::Client.new config
+      @client ||= Msgr::Client.new routes, config
+    end
+
+    def routes
+      @routes ||= Msgr::Routes.new
```

Comment:
Not sure if we want to make this globally accessible - going through `Msgr.client.routes` is a) possible and b) wrapped in a `mutex.synchronize` block.

Reply:
This is now obsolete - this was from a previous state, before I encapsulated the `drain` logic in existing classes.